src/worker/core/draft/genPlayersWithoutSaving.ts

```diff
@@ -67,14 +66,15 @@ const genPlayersWithoutSaving = async (
 		default: 0.5,
 	});
 
-	for (let i = 0; i < 4; i++) {
+	const minMaxAgeDiff = draftAge[1] - draftAge[0];
+	for (let i = !isSport("football") ? 0 : -2; i < minMaxAgeDiff + 1; i++) {
```

Comment:
I think it might be clearer to split this into two loops. One just for football (develop everybody 2 seasons inside an `if (isSport("football")`), and then one to do the rest. I think that would more clearly show how FBGM is being treated special. Like currently you might have to stare at this a while to realize that the `i >= 0` condition can only ever be false for FBGM.